android/src/main/java/com/twiliovoicereactnative/IncomingCallNotificationService.java

```diff
@@ -173,6 +175,13 @@ private void handleIncomingCall(CallInvite callInvite, int notificationId, Strin
     LocalBroadcastManager.getInstance(this).sendBroadcast(intent);
   }
 
+  private void startForegroundCompat(int id, Notification notification) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+      startForeground(id, notification, ServiceInfo.FOREGROUND_SERVICE_TYPE_MICROPHONE);
```

Comment:
wow ... so specific.

Reply:
lol.. yup thats the new android addition they require.. I'm guessing for power savings reasons.?